### PR TITLE
Feat/page materiały dodatkowe

### PR DESCRIPTION
**Pull request by AW2077**
jak w tytule

(((tak, widzę, że zrobiłam checkout ze złego brancha, będę uważać następnym razem)))

### CODE REVIEW

File: src/app/start/StartPage.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import Card from "../../components/Card/Card";
+import styles from "./StartPage.module.scss";
+
+export default function StartPage() {
+  return (
+    <div className={styles.pageWrapper}>
+      <h1 className={styles.heading}>Witaj na Lesson4Future!</h1>
+      <section id="start-menu" className={styles.containerRows}>
+        <StartButton
+          iconClass="fa-solid fa-gamepad"
```
Comment (szymonkadas) [inline src/app/start/StartPage.tsx:12]: nit: please use camelCase for classes, i know that css has standard of kebab-case but due to using mostly css modules we should keep consistency.

File: src/app/Routing.tsx [hunk 2]
```diff
@@ -20,6 +22,9 @@ const Routing: React.FC = () => {
       <Route path="/login" element={<LoginPage />} />
       <Route path="/register" element={<LoginPage />} />
       <Route path="/404" element={<NoMatch />} />
+
+      <Route path="/start" element={<StartPage />} />
+      <Route path="/dodatkowe" element={<AdditionalPage />} />
```
Comment (szymonkadas) [inline src/app/Routing.tsx:27]: We should discuss path naming whether it should be in english or polish.

**Review by szymonkadas — APPROVED** [review]
Overally good job, one nitpick to resolve before merging

File: src/app/Routing.tsx (hunk 2 above)
Comment (AW2077) [reply src/app/Routing.tsx:27]: agree, lets save it for monday

**Pull request merged**